### ISSUE TITLE
Drag and drop stops working randomly

### ISSUE THREAD

**Issue by eternal-sorrow** [swaywm/sway#4235]
I thought this issue was fixed long ago. But I just got it again with sway 1.1.1. Without any apparent reason I was unable to rearrange tabs in Firefox (running in wayland natively) by dragging them. I also opened a Nemo file manager window to test it and yes, drag and drop does not work in it too.

**Comment by luispabon** [comment]
* Sway Version:
1.1.1 (reported by sway as 1.1-rc1-35-g7a11b58a for some reason, but it's built from the 1.1 branch)
wlroots 0.6.0
nautilus 3.32.1 (Ubuntu 19.04)
firefox nightly (on wayland)

* Debug Log:
When trying to drag a file in Nautilus
```
2019-06-24 10:48:52 - [types/data_device/wlr_data_offer.c:87] Ignoring wl_data_offer.accept request on a non-drag-and-drop offer
2019-06-24 10:48:52 - [types/wlr_surface.c:609] New wlr_surface 0x5559590851e0 (res 0x555959038b40)
2019-06-24 10:48:52 - [types/seat/wlr_seat_pointer.c:418] Pointer grab serial validation failed: button_count=5 grab_serial=90 (got 4873)
2019-06-24 10:48:52 - [types/seat/wlr_seat_touch.c:371] Touch grab serial validation failed: num_points=0 grab_serial=0 (got 4873)
2019-06-24 10:48:52 - [sway/input/seat.c:349] Ignoring start_drag request: could not validate pointer or touch serial 4873
```

When trying to drag a tab within a firefox window
```
2019-06-24 10:55:11 - [types/wlr_surface.c:609] New wlr_surface 0x555959041b00 (res 0x55595904c270)
2019-06-24 10:55:11 - [types/seat/wlr_seat_pointer.c:418] Pointer grab serial validation failed: button_count=5 grab_serial=90 (got 6259)
2019-06-24 10:55:11 - [types/seat/wlr_seat_touch.c:371] Touch grab serial validation failed: num_points=0 grab_serial=0 (got 6259)
2019-06-24 10:55:11 - [sway/input/seat.c:349] Ignoring start_drag request: could not validate pointer or touch serial 6259
```
* Configuration File:

https://github.com/luispabon/sway-dotfiles/tree/master/configs/sway

Looks similar to https://github.com/swaywm/sway/issues/3801. Doesn't look like that commit is on my build though but some of the code has changed so much since then I wouldn't be able to patch 1.1 with it.

Long story short, drag & drop will stop working at some point. Sometimes on a fresh sway session, sometimes a couple of days into a session.

**Comment by compilenix** [comment]
I've got the same issue with `sway-1.5-3.fc33.src.rpm`. Several hours later, without restarting sway, drag and drop starts working again and somtime later it stops working again.

My sway sessions are usually running for one to three months at a time.

**Comment by uhetz** [comment]
I'm getting that issue too.

`sway 1:1.5.1-1`

**Comment by pineman** [comment]
also experiencing this on arch linux, sway 1.6

**Comment by vncsna** [comment]
I'm getting the same issue on manjaro with sway 1.6.1. On PCManFM (to PCManFM) I can drag and drop a file only one time, then it's disabled.

**Comment by compilenix** [comment]
I experience the same issue still in version `sway-1.8.1-1.fc38`

**Comment by madeleine313** [comment]
Drag and drop also randomly stops working for me, sway 1.10.

**Comment by eternal-sorrow** [comment]
This one is fixed long ago, if you experience this on a new version, please open a new issue.
